src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import core.basesyntax.lotteryball.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[3];
+
+        // Loop for easier out puts objects accessible by lottery array
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery();
+            System.out.println(lottery[i].getRandomBall().toString());
```

Comment:
we dont need to call toString() method in System.out.println, it will be called "under the hood"